test/api/test_deskew_api.py

```diff
@@ -0,0 +1,161 @@
+#  coding: utf-8
+#  ----------------------------------------------------------------------------
+#  <copyright company="Aspose" file="test_deskew_api.py">
+#    Copyright (c) 2019 Aspose Pty Ltd. All rights reserved.
+#  </copyright>
+#  <summary>
+#    Permission is hereby granted, free of charge, to any person obtaining a
+#   copy  of this software and associated documentation files (the "Software"),
+#   to deal  in the Software without restriction, including without limitation
+#   the rights  to use, copy, modify, merge, publish, distribute, sublicense,
+#   and/or sell  copies of the Software, and to permit persons to whom the
+#   Software is  furnished to do so, subject to the following conditions:
+#
+#   The above copyright notice and this permission notice shall be included in
+#   all  copies or substantial portions of the Software.
+#
+#   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+#   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+#   AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+#   FROM,  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+#   DEALINGS IN THE SOFTWARE.
+#  </summary>
+#  ----------------------------------------------------------------------------
+
+from itertools import product
+
+import asposeimagingcloud.models.requests as requests
+from test.api.imaging_api_tester import ImagingApiTester
+
+
+class TestDeskewApi(ImagingApiTester):
+    """ Class for testing DeskewAPI """
+
+    def test_deskew_image(self):
+        """ Test deskew_image """
+
+        if not self.EXTENDED_TEST:
+            format_extension_test_cases = ['.jpg']
+        else:
+            format_extension_test_cases = [
+                '.jpg',
+                '.bmp',
+                #'.dicom',
```

Comment:
@dmitrypasyutin Please, provide a comment with the reason why some of the test cases are commented out, so we could enable them when the appropriate feature is implemented on the server side, for example.